README.md

```diff
@@ -0,0 +1,17 @@
+# 3p-SDL
+
+This is a fork of [Alchemy's](https://www.alchemyviewer.org/) [3p-SDL2](https://git.alchemyviewer.org/alchemy/thirdparty/3p-sdl2)
+repository configured to build and package using [autobuild](https://wiki.secondlife.com/wiki/Autobuild).
+
+[Simple DirectMedia Layer](SDL/docs/README.md) is a cross-platform development library designed
+to provide low level access to audio, keyboard, mouse, joystick, and graphics
+hardware via OpenGL and Direct3D.
+
+# How to build:
+
+* Install [autobuild](https://wiki.secondlife.com/wiki/Autobuild).
+* <pre>autobuild configure</pre>
+* <pre>autobuild build</pre>
+* <pre>autobuild package</pre>
```

Comment:
Nit: replace use of `<pre>` with [backticks](https://www.markdownguide.org/basic-syntax/#code).